### DISCUSSION TITLE
Запретить создавать таблицу "Расчетная в цикле"

### DISCUSSION THREAD

**Discussion by simplepersonru** [totumonline/totum-ru-issues-and-discussions#414] [Есть идея, а что если...]
При условии, если пытаемся создать таблицу просто в разделе. Это не имеет смысла и чота ломает. Короче вредно. А еще она более заметная чем "Циклов" и поэтому иногда очень хочется ее нажать, когда хочу создать таблицу "Циклов"...
Возможно это касается и "Расчетная в дереве", но с такими пока еще не работал
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/71624171/8917a69c-ac68-412f-bc8f-a34dbe3cd13d)

**Comment by ttm-alexeypolunin** [comment]
Да - это место надо улучшить. Когда-нибудь оно улучшится.
